crypto/fipsmodule/ec/ec.c

```diff
@@ -1152,12 +1152,6 @@ int EC_METHOD_get_field_type(const EC_METHOD *meth) {
 
 void EC_GROUP_set_point_conversion_form(EC_GROUP *group,
                                         point_conversion_form_t form) {
-  /* NO-OP. However, abort if consumer attempts to set a representation that is
-   * not supported. */
-  if (form != POINT_CONVERSION_UNCOMPRESSED &&
-      form != POINT_CONVERSION_COMPRESSED) {
```

Comment:
why not just add `&& form != POINT_CONVERSION_HYBRID`?

Reply:
We're supporting all possible representations now, so I thought it made sense to just remove it since the comment above was no longer applicable. I could add it back if we really wanted to?